cornice_swagger/swagger.py

```diff
@@ -498,7 +501,8 @@ def get_tags_from_path(service):
 
     swag = CorniceSwagger(services, def_ref_depth=-1, param_ref=0)
     doc = swag(title, version, ignores=('head'),
-               default_tags=get_tags_from_path, **kwargs)
+               default_tags=get_tags_from_path,
+               summary_docstrings=True, **kwargs)
     doc.update(**kwargs)
```

Comment:
Hmm, I just noticed this.. this means that e.g. `"summary_docstrings": true` could show up in a Swagger spec. This seems bad to me.